examples/seq2seq/seq2seq.py

```diff
@@ -371,6 +371,10 @@ def translate(trainer):
             trigger=(args.validation_interval, 'iteration'))
 
     print('start training')
+    if args.resume:
+        # Resume from a snapshot
+        chainer.serializers.load_npz(args.resume, trainer)
```

Comment:
It seems `save_npz` or snapshot extension is not used in this example.
Could you add?

Reply:
Done.